app/services/legacy/dataset_import_service.rb

```diff
@@ -209,10 +207,8 @@ def calculate_dates_for_year(year)
     "31/12/#{year}"
   end
 
-  private
-
   def legacy_datafiles
-    resources.select{ |resource| resource['resource_type'] == 'file'}
+    resources.select{ |resource| resource['resource_type'] != 'documentation'}
```

Comment:
Instead of a double negative you could change this to use reject:

```ruby
resources.reject { |resource| resource['resource_type'] == 'documentation' }
```